pkg/virt-controller/services/template.go

```diff
@@ -150,6 +193,22 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 	resources := k8sv1.ResourceRequirements{}
 	vmiResources := vmi.Spec.Domain.Resources
 
+	isMemoryFiledExist := func(resource k8sv1.ResourceList) bool {
```

Comment:
Field? was this supposed to be named `isMemoryFieldExist`

Reply:
Yup, thanks :)